.github/workflows/deploy.yaml

```
@@ -27,7 +27,7 @@ jobs:
     - name: Build
       run: |
         cd app
-        yarn build
+        yarn build:devnet
     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)